frontend/src/index.jade

```diff
@@ -148,6 +153,8 @@ html
             .title(onclick="toggleNext(this)")
               span {{ file.path }}&nbsp;
               span.loc ({{ file.lineCount }} lines)
+            #downloadCheckbox
+              input(type="checkbox", :value="file", v-model="filesDownloaded")
```

Comment:
shouldn't `filesDownloaded` be `filesSelected`?

Reply:
Are you referring to the naming of variable? `filesSelected` may conflict with PR #448 if merged.

Reply:
Yes. #448 can probably be `fileTypeSelected`? So there will be no conflict. 

Also, just a correction, since the value is `file`, it should correspondingly be `file...`